web_widget_image_paint/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Copyright 2009-2019 Noviat.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    'name': 'Image Paint widget',
+    'version': '12.0.1.0.0',
+    'author': 'Noviat',
+    'website': 'http://www.noviat.com',
```

Comment:
@vanderperre the website should be 'https://github.com/OCA/web'

Reply:
ok, i will commit it now as -m 'the website should be oca'.
3ad0d4cb..ca5adc33  web_widget_image_paint -> web_widget_image_paint